.hlxignore

```diff
@@ -1,4 +1,10 @@
 .*
 *.md
+*.json
+!*/**/*.json
 LICENSE
 test/*
+*.mjs
```

Comment:
You potentially want a similiar modifier `!*/**/*.mjs` to keep certain mjs modules hidden within the repo to avoid weird side effects in the future